package.json

```diff
@@ -17,10 +17,11 @@
     ]
   },
   "scripts": {
-    "build": "npm run build --workspace=@faustjs/core --workspace=@faustjs/react --workspace=@faustjs/next",
+    "build": "npm run build --workspace=@faustjs/core --workspace=@faustjs/react --workspace=@faustjs/next --workspace=faust-nx",
     "build:core": "npm run build --workspace=@faustjs/core",
     "build:next": "npm run build --workspace=@faustjs/next",
     "build:react": "npm run build --workspace=@faustjs/react",
+    "build:faustnx": "npm run build --workspace=faust-nx",
     "clean": "npm run clean --workspace=@faustjs/core --workspace=@faustjs/react --workspace=@faustjs/next",
     "clean:examples": "rimraf examples/**/node_modules",
     "lint": "npm run lint --workspace=@faustjs/core --workspace=@faustjs/react --workspace=@faustjs/next",
```

Comment:
Let's hold off on this one since at the package level we don't yet have an ESLint config. The work @TeresaGobble is doing will properly setup linting for the FaustNX package.